README.md

```diff
@@ -12,6 +12,29 @@ This extension adds advanced support for Elixir features. It is powered by elixi
 - Format code on save (experimental)
 - Find references (experimental)
 
+## Contributions
+
+Help improving this extension.
+
+To do this, you need to execute the following commands.
+
+```shell
+pwd # should look like /somewhere/on/your/drive/in/nova-elixir-ls
+cd elixir-ls
+mix deps.get
+mix compile
+mix elixir_ls.release o- ../ElixirLS.novaextension/elixir-ls-release
+cd ...
+```
+
+Then copy the content of the folder 'src'.
+
+```shell
+cp -r src/ ElixirLS.novaextension/Scripts/
```

Comment:
Instead of copying the `src` folder we should encourage people to use the, newly added, `build` command.
```
npm install
npm run build
```
or
```
yarn install
yarn run build
```

Reply:
Good point.